de.sesu8642.feudaltactics.yaml

```
@@ -0,0 +1,40 @@
+app-id: de.sesu8642.feudaltactics
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.openjdk17]
```

```suggestion
sdk-extensions: 
  - org.freedesktop.Sdk.Extension.openjdk17
```

let's write clean YAML